bot/services/tag_service.py

```diff
@@ -19,8 +19,11 @@ class TagService(BaseService):
     def __init__(self, *, bot):
         super().__init__(bot)
     
-    @BaseService.Listener(Events.on_message_recieved)
-    async def on_message_recieved(self, message: discord.Message) -> None:
+    @BaseService.Listener(Events.on_message_received)
```

Comment:
Need to update to the new events and verify no other locations are still referencing this old event.